.travis.yml

```diff
@@ -306,6 +306,11 @@ jobs:
       if: type = pull_request
       php: 7.2
       script:
+        - |
```

Comment:
Should be gone after a rebase, no?

Reply:
I’ll remove it manually if it doesn’t. I tested the fix here. Gonna rebase develop and this branch tomorrow.